src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,37 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private String bulldozerChassisType;
```

Comment:
The fields `bulldozerChassisType`, `bladeType`, and `bulldozerType` should have access modifiers to encapsulate the fields properly. Consider using private access modifiers and providing public getter and setter methods.